src/k2/dto/K23SI.h

```diff
@@ -219,6 +219,32 @@ struct K23SIWriteResponse {
     K2_PAYLOAD_EMPTY;
 };
 
+struct K23SIPartialUpdateRequest{
+    Partition::PVID pvid; // the partition version ID. Should be coming from an up-to-date partition map
+    String collectionName; // the name of the collection
+    K23SI_MTR mtr; // the MTR for the issuing transaction
+    // The TRH key is used to find the K2 node which owns a transaction. It should be set to the key of
+    // the first write (the write for which designateTRH was set to true)
+    // Note that this is not an unique identifier for a transaction record - transaction records are
+    // uniquely identified by the tuple (mtr, trh)
+    Key trh;
+    bool isDelete = false; // is this a delete write?
```

Comment:
Delete should be just a WriteRequest and partialUpdate doesn't need to support it, right?

Reply:
yes, I agree